agent.md

```
@@ -0,0 +1,1268 @@
+```
+
+### Frontend (`.env.local`)
+
+```bash
+NEXT_PUBLIC_API_BASE=http://localhost:4000/api
+NEXT_PUBLIC_ENABLE_OBSERVABILITY=true
+```
+
+### Supabase Setup
+
+1. Create project at https://supabase.com
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)